R/process_lookup_sc_client.R

```diff
@@ -150,6 +150,9 @@ process_lookup_sc_client <-
       )) == "Not Known")) %>%
       dplyr::arrange(chi, count_not_known) %>%
       dplyr::distinct(chi, .keep_all = TRUE) %>%
+      dplyr::mutate(
+        sc_send_lca = convert_sc_sending_location_to_lca(sending_location)
+      ) %>%
```

Comment:
This can be removed as this will introduce duplicate code. We previously fixed this in this PR https://github.com/Public-Health-Scotland/source-linkage-files/pull/881/files